core/state_transition.go

```diff
@@ -646,6 +641,12 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 		}, nil
 	}
 
+	if rules.IsOptimismIsthmus {
```

Comment:
Let's just add a small comment above for posterity.

```suggestion
	// Operator Fee refunds are only applied if isthmus is active and the transaction is *not* a deposit.
	if rules.IsOptimismIsthmus {
```